frontend/packages/console-shared/src/components/terminal/TerminalDrawer.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { DraggableData } from 'react-draggable';
+import { Tooltip, Button, Flex, FlexItem, Bullseye, FlexModifiers } from '@patternfly/react-core';
+import {
+  MinimizeRestoreButton,
+  MinimizeRestoreButtonState,
+  CloseButton,
+  ExternalLinkButton,
+} from '../buttons';
+import { Drawer, DrawerHeader } from '../drawer';
+import './TerminalDrawer.scss';
+
+const INITIAL_HEIGHT = 318;
+
+const TerminalDrawer: React.FC = ({ children }) => {
+  const [open, setOpen] = React.useState(false);
+  const [height, setHeight] = React.useState(318);
+  const [minHeight, setMinHeight] = React.useState(0);
+  const [minimized, setMinimized] = React.useState(MinimizeRestoreButtonState.minimize);
+  const [lastObservedHeight, setLastObservedHeight] = React.useState(318);
+  const handleResize = (e: MouseEvent, data: DraggableData) => {
+    setHeight(height - data.y);
+  };
+  const handleResizeStart = (e: MouseEvent, data: DraggableData) => {
+    setLastObservedHeight(height - data.y);
+  };
+  const handleResizeStop = () => {
+    if (height <= minHeight) {
+      setMinimized(MinimizeRestoreButtonState.restore);
+    }
+    if (height > minHeight) {
+      setMinimized(MinimizeRestoreButtonState.minimize);
+      setLastObservedHeight(height);
+    }
+  };
+  const onMRButtonClick = (state: MinimizeRestoreButtonState) => {
+    if (state === MinimizeRestoreButtonState.minimize) {
+      setMinimized(MinimizeRestoreButtonState.restore);
+      setHeight(minHeight);
+    }
+    if (state === MinimizeRestoreButtonState.restore) {
+      setMinimized(MinimizeRestoreButtonState.minimize);
+      setHeight(lastObservedHeight);
+    }
+  };
+
+  const handleCloseButton = () => {
+    setOpen(false);
+    setHeight(INITIAL_HEIGHT);
+  };
+  return (
+    <>
+      <Button variant="control" onClick={() => setOpen(!open)}>
+        Open Drawer
+      </Button>
+      <Drawer
+        toggle={open}
+        height={height}
+        minHeight={minHeight}
+        onResize={handleResize}
+        onResizeStart={handleResizeStart}
+        onResizeStop={handleResizeStop}
+        isResizable
+      >
+        <DrawerHeader className="ocs-terminal-drawer__header">
+          {(size) => {
+            if (size.height) {
+              setMinHeight(size.height);
+            }
+            return (
+              <Flex>
+                <FlexItem>
+                  <Bullseye className="ocs-terminal-drawer__heading">
+                    Command Line Terminal
+                  </Bullseye>
```

Comment:
You do not need a `Bullseye` to center align the text. Use flex alignment css properties.